File: src/nl/rubensten/texifyidea/action/insert/InsertUnderlineAction.java
package nl.rubensten.texifyidea.action.insert;

import nl.rubensten.texifyidea.TexifyIcons;
import nl.rubensten.texifyidea.action.InsertEditorAction;

/**
 * @author Ruben Schellekens
 */
public class InsertUnderlineAction extends InsertEditorAction {

    public InsertUnderlineAction() {
        super("Underline", TexifyIcons.FONT_UNDERLINE, "\\underline{", "}");
    }
}

File: src/nl/hannahsten/texifyidea/action/insert/InsertUnderlineAction.kt
package nl.hannahsten.texifyidea.action.insert;

import nl.hannahsten.texifyidea.TexifyIcons;
import nl.hannahsten.texifyidea.action.InsertEditorAction;

/**
 * @author Hannah Schellekens
 */
public class InsertUnderlineAction extends InsertEditorAction {

    public InsertUnderlineAction() {
        super("Underline", TexifyIcons.FONT_UNDERLINE, "\\underline{", "}");
    }
}
